eks-anywhere-common/Testers/Sysdig/sysdig-test-job.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
```

Comment:
This should be replaced to a CronJob which runs on Schedule?

Reply:
Daily Cronjob has been implemented